torch/csrc/jit/script/init.cpp

```diff
@@ -745,12 +782,12 @@ void initJitScriptBindings(PyObject* module) {
           })
       .def(
           "_python_print",
-          [](Module& self) {
+          [](Module& self, bool enforce_importable) {
             std::ostringstream ss;
             std::vector<at::Tensor> tensors;
-            PythonPrint(ss, self, tensors, true);
+            PythonPrint(ss, self, tensors, enforce_importable);
             return std::make_pair(ss.str(), tensors);
-          })
+          }, py::arg("enforce_importable") = true)
```

Comment:
Why do we need this?

Reply:
Mostly just for the test added in `test_jit`, plus if someone wants to easily see what exported code will look like then this flag is necessary

Reply:
I can't see any references to this in the tests and I still don't fully understand when would one set this to false. It would print Python ops as `torch.ops.prim.PythonOp()`, right?

Reply:
Revisiting this I'm not sure why this was added..., it's removed in the latest commit